cmd/gravity/Cargo.toml

```
@@ -10,20 +10,16 @@ repository = "https://github.com/arcjet/gravity"
 description = """
 Gravity is a host generator for WebAssembly Components. It currently targets Wazero, a zero dependency WebAssembly runtime for Go.
 """
-build = "build.rs"
 
 [[bin]]
 name = "gravity"
 path = "src/main.rs"
 
 [dependencies]
 clap = "=4.5.47"
-genco = "=0.17.10"
+genco = "0.18"
```

```suggestion
genco = "=0.18.1"
```
Let's keep this pinned since lockfiles aren't applied when installing packages from `crates.io`